CHANGELOG.md

```diff
@@ -67,6 +67,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added doc on `cluster create-index blocked` workaround ([#465](https://github.com/opensearch-project/opensearch-api-specification/pull/465))
 - Added support for reusing output variables as keys in payload expectations ([#471](https://github.com/opensearch-project/opensearch-api-specification/pull/471))
 - Added support for running tests against Amazon OpenSearch ([#476](https://github.com/opensearch-project/opensearch-api-specification/pull/476))
+- Added tests for security APIs ([#439](https://github.com/opensearch-project/opensearch-api-specification/pull/439))
```

Comment:
No need to mention tests as they are not external.

Mention additions here (e.g. "Added `/_plugins/_security/api/certificates/{node_id}`"), and fixes under fixes with some specifics (e.g. "Fixed `/_plugins/_security/api/_upgrade_check`").

Reply:
updated changelog. I've also added an entry for specs that I added originally.